app/app/routes.py

```diff
@@ -22,4 +22,12 @@ def data():
     if current_user.is_authenticated:
         user = User.query.filter_by(id = current_user.id).first()
 
-    return render_template('data.html', user=current_user)
\ No newline at end of file
+    return render_template('data.html', user=current_user)
+
+@routes.route('/export')
+@login_required
+def export():
+    if current_user.is_authenticated:
+        user = User.query.filter_by(id = current_user.id).first()
```

Comment:
This is an unused variable?

Reply:
I'm not too sure about this. I just copied the same validation code that is used on the other pages. @JordanHartley Do you know why user is an unused variable?

Reply:
As @JordanHartley mentioned, user is indeed an unused variable. However, it may be useful if in the future multiple user roles are implemented

Reply:
okay your choice, but I would either put a comment there as "TODO" or remove it, as it add unnecessary confusion or ambiguity